app/models/recurring_todos/abstract_repeat_pattern.rb

```diff
@@ -174,7 +174,7 @@ def continues_recurring?(previous)
     # same day as the previous
     def determine_start(previous, offset=0.day)
       start = self.start_from || NullTime.new
-      now = Time.zone.now
+      now = Time.now
```

Comment:
I'm not convinced by my fix either, so I'm happy to discuss about this :)
What I think is that in the DB we are saving all the dates without any TZ info. So I think that we should just translate to the relevant TZ when we display the data to the user.
From what I can see in this particular test, the date in the DB is saved in the UTC TZ, even if I start with the enviornment variable TZ set to something else. Is that a reliable way to do it? And if the data in the DB is without TZ info then we should compute everything accordingly, am I understanding right?